pbft/pbft.hpp

```diff
@@ -84,8 +88,16 @@ namespace bzn
 
         std::string get_name() override;
 
+        bool is_view_valid() const;
+
+        uint64_t get_view() const { return this->view; }
```

Comment:
This is the only inline method. Consider moving implementation to .cpp file.

Reply:
Done. Moved implementation to cpp.